.github/workflows/pnpm_docker_image_from_config.yml

```diff
@@ -0,0 +1,264 @@
+name: Build and Push Docker Images on release
```

Comment:
Add a better description what this actually does and why is it different from pnpm_docker_umage.yml

(I know, others do not)